pynecone/.templates/app/tutorial.py

```diff
@@ -1,3 +1,8 @@
+# type: ignore
```

Comment:
Hmm this is a good warning but I also don't really want all this text at the top to scare off new users and to keep the template file clean. Maybe instead we can print some warning during `pc run` that tells them not to worry about the warning or something?